exp-player/config/environment.js

```diff
@@ -2,5 +2,32 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+      moment: {
+          includeLocales: true
```

Comment:
I see this was moved back to code review, but this comment was unaddressed. Can you comment on the decision to leave as is? Are there technical barriers I can help with?